_posts/2024-08-13-#3179.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: pr
-date: 2024-08-13
+date: 2024-10-11
```

Comment:
Isn't the review club today for 3010? Looks like a few of them might be wrong. Additionally, do we need to change file names for this to show up correctly on the web site?

Reply:
Oh we have the date also inside the name :) thanks.

And yes we may have the date wrong in other places I will take a look at them after today review meeting, and maybe make a followup PR